### PR TITLE
Add API documentation for confirmed_at

### PR DESCRIPTION
**Pull request by martijnbleeker**
Can be merged when https://github.com/q-m/questionmark-api/pull/2420 is deployed.

### CODE REVIEW

File: api-docs/swagger-v1.2.yml [hunk 1]
```diff
@@ -130,6 +130,7 @@ paths:
         - $ref: '#/parameters/package_size_min'
         - $ref: '#/parameters/package_size_max'
         - $ref: '#/parameters/package_size_unit'
+        - $ref: '#/parameters/maximum_age'
```
Comment (wvengen) [inline api-docs/swagger-v1.2.yml]: We have min/max as a suffix already, what about `confirmed_age_max` or so?

**Review by wvengen — COMMENTED** [review]
One question, otherwise fine!
Please do mention in the PR description that this can only be merged if PR... is deployed to production.

File: api-docs/swagger-v1.2.yml (hunk 1 above)
Comment (martijnbleeker) [reply api-docs/swagger-v1.2.yml]: But then the documentation would be incorrect. 😏

Comment (martijnbleeker) [reply api-docs/swagger-v1.2.yml]: Changed it, it doesn't sound correct in my head though, maybe something to consider for API 2.0

Comment (wvengen) [reply api-docs/swagger-v1.2.yml]: Hah :) I'll put it in the code PR.

Comment (wvengen) [reply api-docs/swagger-v1.2.yml]: ... but wait, you're blazingly fast :o

**Comment by wvengen** [comment]
I think we can still merge this, right?

**Pull request closed without merging**